test/extended/deployment_test.go

```diff
@@ -0,0 +1,69 @@
+package extended
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	log "github.com/sirupsen/logrus"
+)
+
+var _ = g.Describe("Deployment Suite", func() {
+	defer g.GinkgoRecover()
+
+	// Create temp namespace based on ginko seed
+	g.BeforeEach(func() {
+		// Create temp namespace for tests
+		buf, err := Asset("Namespaces/test.yaml")
+		o, err := cl.Unmarshal(buf)
```

Comment:
I'm not liking this pattern.  I'd be in favour of having `cl` include a proper k8s clientset; separate free-standing helper functions to create and tear down namespaces, no yaml files.